packages/vscode-docker-registries-extension/src/extension.ts

```diff
@@ -23,6 +23,7 @@ export function activate(context: vscode.ExtensionContext) {
 	context.subscriptions.push(vscode.commands.registerCommand('dockerRegistries2.refreshRegistries', () => urtdp.refresh()));
 	context.subscriptions.push(vscode.commands.registerCommand('dockerRegistries2.connectRegistry', () => urtdp.connectRegistryProvider()));
 	context.subscriptions.push(vscode.commands.registerCommand('dockerRegistries2.disconnectRegistry', (item) => urtdp.disconnectRegistryProvider(item)));
+	context.subscriptions.push(vscode.commands.registerCommand('dockerRegistries2.disconnectGenericV2Registry', (item) => urtdp.disconnectRegistryProvider(item)));
```

Comment:
There needs to be a corresponding connect generic V2 registry command so that you can connect to multiple at the same time.

Reply:
It should call into `addTrackedRegistry`.

Reply:
ahh yeah that makes sense